package.json

```diff
@@ -14,7 +14,7 @@
   "homepage": "https://github.com/juliuste/parliament-svg",
   "repository": "juliuste/parliament-svg",
   "bugs": "https://github.com/juliuste/parliament-svg/issues",
-  "main": "./index.js",
+  "main": "./dist/index.js",
```

Comment:
It would make sense to add another field to indicate the location of untranspiled code, for usage with ES6-aware bundlers, Node.js, etc. [`module` seems to be the new standard for this](https://github.com/jsforum/jsforum/issues/5).

Note that this is a regression in some way: Node.js will now run (the transpiled) `dist/index.js` instead of (the untranspiled) `lib/index.js`. I'd therefore propose bumping the major version.